shakenfist/daemons/daemon.py

```diff
@@ -0,0 +1,25 @@
+import setproctitle
+
+from shakenfist import util
+
+
+DAEMON_NAMES = {
+    'API': 'sf-api',
+    'CLEANER': 'sf-cleaner',
+    'MAIN': 'sf-main',
+    'NET': 'sf-net',
+    'RESOURCES': 'sf-resources',
+    'TRIGGERS': 'sf-triggers',
+}
```

Comment:
This is nice, although I find the upper case here jarring on reflection.

Reply:
Certainly can change it. Lower case? 
I was copying code like `config.parsed.get('NODE_NAME')`

Reply:
Yeah, the config thing is using environment variable naming style, whereas these are just strings. I think lowercase is less jarring to be honest, but I know its a nit.